### PR TITLE
refactor, add links

### PR DESCRIPTION
**Pull request by WalterBright**
And since gawd hath invented alpha sort, so shall it be done, and it was good.

Added navigation links.

Converted tabs to spaces.

No text changes.

### CODE REVIEW

File: pragma.dd [hunk 1]
```diff
@@ -57,11 +57,19 @@ $(H3 $(LEGACY_LNAME2 Predefined-Pragmas, predefined-pragmas, Predefined Pragmas)
 
         $(P All implementations must support these, even if by just ignoring
         them:
+
+        $(UL
+            $(LI $(LINK2 #inline, pragma inline))
```
Comment (andralex) [inline pragma.dd]: Need `$(HASH)` instead of `#`. Run `make pdf` to make sure the pdf makes it through.

**Comment by CyberShadow** [comment]
Looks fine on the doc tester:

http://dtest.thecybershadow.net/artifact/website-9bbe7b16aaa20f28b879722609308a024a12ae52-88b9585ccbd203d7a6fba092a71f9acd/web/dlangspec.pdf (page 128)

**Comment by andralex** [comment]
@CyberShadow how come latex didn't choke on the pound sign?

**Comment by andralex** [comment]
apparently it works! pulling

**Pull request merged**